0x/Orders/Order.cs

```diff
@@ -0,0 +1,66 @@
+﻿using EIP712;
+using System;
+using System.Numerics;
+using ZeroX.Assets;
+using ZeroX.Utilities;
+
+namespace ZeroX.Orders
+{
+    public class Order
+    {
+        // TODO: Add docs, property validation
+
+        public EthereumAddress MakerAddress { get; set; }
+        public EthereumAddress TakerAddress { get; set; }
+        public EthereumAddress FeeRecipientAddress { get; set; }
+        public EthereumAddress SenderAddress { get; set; }
+        public BigInteger MakerAssetAmount { get; set; }
+        public BigInteger TakerAssetAmount { get; set; }
+        public BigInteger MakerFee { get; set; }
+        public BigInteger TakerFee { get; set; }
+        public BigInteger ExpirationTimeSeconds { get; set; }
+        public BigInteger Salt { get; set; }
```

Comment:
Is this necessary a number?

Reply:
Salt*

Reply:
Salt is BigNumber because corresponding member in solidity struct is of type "uint256"